wca-regulations.md

```diff
@@ -358,13 +358,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 ## <article-A><speedsolving><speedsolving> Article A: Speed Solving
 
 - A1) Speed Solving attempts must abide by the following procedures.
-    - A1a) The organization team must enforce time limits for solves and/or rounds.
-        - A1a1) The default time limit per solve is 10 minutes, though the organization team may announce a higher or lower time limit.
-        - A1a2) Cumulative time limits may be enforced (e.g. 3 solves with a cumulative time limit of 20 minutes - see [Regulation A1a5](regulations:regulation:A1a5)). The time elapsed in a DNF result counts towards the cumulative time limit.
-        - A1a3) For each round, any time limits must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
-        - A1a4) The competitor must end each solve within the time limit. If a competitor reaches the time limit for a solve/round, the judge stops the solve immediately and records the result as DNF. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-        - A1a5) A solve is considered to meet the time limit if and only if the final result, after any time penalties are applied, is less than the time limit. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-    - A1b) If the time limit for a solve is greater than 10 minutes, a stopwatch must be used for timekeeping.
+    - A1a) The organization team applies exactly one kind of time limit (time limit per attempt or cumulative time limit) to each round.
```

Comment:
I'm not a native speaker, but the wording seems weird to me. I'd prefer a slight modification on the original one:

The organization team must apply exactly one kind of time limit ...

Reply:
I think "must apply" is actually weirder due to most English speakers, but cubers who read the Regulations are lot are very used to it.

I'd prefer to keep it this way, because it's more concise, and matches a lot of the rest of Article A.

Reply:
>I'd prefer to keep it this way, because it's more concise, and matches a lot of the rest of Article A.

I disagree with that. Article A uses 'must do something' for rules that need to be followed "in general" (see A1a3, A1a4, A1b(1), A1c and others), and uses verbs in present simple for actions to be taken on the spot (see A2a, A2b, A2c1, A2d, etc)

I believe A1a falls into the first category, not the second.

Reply:
I suppose that's a fair argument. I'd be happy to approve and merge a small PR with that change if you'd like!